<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GalleryFrame</class>
 <widget class="QFrame" name="GalleryFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QLabel" name="label_Title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::MarkdownText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Description">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="text">
      <string>A badge can have a filled, ghost, outline, or tint appearance. The default is filled.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::MarkdownText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_Content">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frame_Content{
	background-color: #fafafa;
	border-color: 1px solid #d1d1d1;
	border-radius: 16px;
}</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label_Title</zorder>
  <zorder>frame_Content</zorder>
  <zorder>label_Description</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
